docs/specs/toc.yml

```diff
@@ -1070,26 +1070,6 @@ outputs:
   build.log: |
     ["error","output-path-conflict","Two or more files output to the same path 'docs/toc.json': 'docs/TOC.md', 'docs/TOC.yml'"]
 ---
-# validate bookmarks in toc
```

Comment:
don't remove the whole test, just remove the invalid bookmark reference `title-2`

Reply:
ok